SampleBrowser/SampleBrowser/Extensions/WindowUUID+Extension.swift

```diff
@@ -0,0 +1,23 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+/// A unique identifier for a particular window in Firefox iOS.
+/// On iPad this UUID corresponds to an individual window which
+/// manages its own unique set of tabs. Multiple Firefox windows
+/// can be run side-by-side on iPad (once multi-window is enabled). [FXIOS-7349]
+public typealias WindowUUID = UUID
+
+public extension WindowUUID {
```

Comment:
@thatswinnie I think we can probably just remove this extension from SampleBrowser for now at least (and just use UUID). But either way this is Ok with me.

There might be some additional changes needed later once you start working on the themeing for the toolbar (for example we might need to just use a hardcoded UUID for the default window etc), let me know if you run into any issues there and we can sync up if needed.